ats/job_applications/models.py

```diff
@@ -0,0 +1,26 @@
+from django.db import models
+from model_utils import Choices
+from django.utils.translation import ugettext_lazy as _
+
+
+class JobApplications(models.Model):
```

Comment:
Any model name must be singular, not plural
```suggestion
class JobApplication(models.Model):
```